openshift-ci/README.md

```diff
@@ -0,0 +1,193 @@
+# DevConsole Operator Continuous Integration
+
+DevConsole operator uses [OpenShift CI][openshift-ci] for continuous
+integration.  The Openshift CI is built using
+[CI-Operator][ci-operator].  The configuration is located here:
+http://bit.ly/304kpIo
+
+As part of the continuous integration, there are several
+jobs configured to run against pull requests in GitHub.  The CI jobs
+are triggered whenever there is a new pull request from the team
+members.  It is also triggered when there is a new commit in the
+current pull request.  If a non-member creates the pull request, there
+should be a comment with text as `/ok-to-test` from any member to run
+the CI job.
+
+Here is a high-level schematic diagram of the CI pipeline for an
+end-to-end test:
+
+```
++--------+     +--------+     +--------+
+|        |     |        |     |        |
+|  root  +---->+  src   +---->+  bin   |
+|        |     |        |     |        |
++--------+     +--------+     +---+----+
+                                   |
+    ,------------------------------'
+    |
+    v
++---+----+     +--------+
+|        |     |        |
+| images +---->+ tests  |
+|        |     |        |
++--------+     +--------+
+```
+
+For lint and unit test, the schematic diagram is as follows:
+
+```
++--------+     +--------+     +----------------+
+|        |     |        |     |                |
+|  root  +---->+  src   +---->+ lint/unit test |
+|        |     |        |     |                |
++--------+     +--------+     +----------------+
+```
+
+
+All the steps mentioned here is taking place inside a work namespace.
```

Comment:
Very small nit - "is" should be "are" - thanks!

Reply:
Done. ceba1627b1c5d4a8ced946a3b352b1416f4e6c3a